### DISCUSSION TITLE
How can I include [Oh My Zsh] on update?

### DISCUSSION THREAD

**Discussion by Ioulan** [topgrade-rs/topgrade#361] [Q&A] [answered]
My OMZ installation is located at $XDG_DATA_HOME
```
❯  echo $ZSH
/home/void/.local/share/oh-my-zsh
```
Now this doesn't get included on update but,
when  OMZ is at the default location ($HOME) it gets updated

**Comment by SteveLauC** [comment, marked as answer]
I also found this, see issue https://github.com/topgrade-rs/topgrade/issues/426

I will fix this in the near future:)
